src/generate/apiGroup.ts

```diff
@@ -30,7 +29,16 @@ export class ApiGroup {
   static async init(apiSpecFilePaths: string[], name = ""): Promise<ApiGroup> {
```

Comment:
Feels a bit weird to have `init(array, name)` and `constructor(name, ...array)`. Would be nice if they matched.